Makefile

```diff
@@ -151,6 +151,12 @@ target/openapi-schema-validated:
 	touch target/openapi-schema-validated
 
 
+test-release: export RUSTFLAGS += -D warnings
```

Comment:
This sets it for all dependencies, not just the current workspace.

Do you even need this? `CLIPPY_FLAGS` will be enough, no?

Reply:
Oh, you're right, it does set it for all deps.

Sure, I guess it's not really necessary, since clippy catches regular warnings as well. Will fix.